app/src/main/java/com/kms/cura/view/PatientSignUpActivity.java

```diff
@@ -0,0 +1,102 @@
+package com.kms.cura.view;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.kms.cura.R;
+import com.kms.cura.utils.InputUtils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PatientSignUpActivity extends AppCompatActivity implements TextWatcher {
+    private EditText edtFirstName, edtEmail, edtPassword, edtPasswordReenter;
+    private Button btnRegister;
+    private int count = 0;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_patient_sign_up);
+        initEditText();
+        btnRegister = (Button) findViewById(R.id.btnRegister);
+        btnRegister.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                //Patient Register Function
+            }
+        });
+
+    }
+
+
+    public void initEditText() {
+        edtFirstName = initEditText(edtFirstName, R.id.edtFirstName, this);
+        edtEmail = initEditText(edtEmail, R.id.edtEmail, this);
+        edtPassword = initEditText(edtPassword, R.id.edtPassword, this);
+        edtPasswordReenter = initEditText(edtPasswordReenter, R.id.edtPasswordReenter, this);
+    }
+
+    public EditText initEditText(EditText src, int id, TextWatcher watcher) {
```

Comment:
You don't need to pass EditText src here. Just create a local variable EditText and then return it